cocos/2d/assembler/sprite/sliced.ts

```diff
@@ -28,16 +28,21 @@
  * @module ui-assembler
  */
 
-import { Mat4, Vec3 } from '../../../core/math';
+import { Color, Mat4, Vec3 } from '../../../core/math';
 import { IRenderData, RenderData } from '../../renderer/render-data';
 import { IBatcher } from '../../renderer/i-batcher';
 import { Sprite } from '../../components';
 import { IAssembler } from '../../renderer/base';
 import { dynamicAtlasManager } from '../../utils/dynamic-atlas/atlas-manager';
 import { StaticVBChunk } from '../../renderer/static-vb-accessor';
+import zlib from '../../../../external/compression/zlib.min';
 
 const vec3_temp = new Vec3();
 const matrix = new Mat4();
+const diagonal_vec3_temps: IRenderData[] = [];
```

Comment:
name is too long, please modify to a short one, like `tempData`

Reply:
Modified to `tempRenderData`